src/coreclr/tools/aot/ObjWriter/objwriter.cpp

```diff
@@ -607,6 +607,13 @@ void ObjectWriter::EmitVarDefRange(const MCSymbol *Fn,
 // Maps an ICorDebugInfo register number to the corresponding CodeView
 // register number
 CVRegNum ObjectWriter::GetCVRegNum(ICorDebugInfo::RegNum RegNum) {
+  static const CVRegNum cvRegMapAmd64[] = {
```

Comment:
Nit: They also start variable names with a capital letter: https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly. Otherwise looks great.